tools/functional-tester/etcd-tester/tester.go

```diff
@@ -174,21 +170,25 @@ func (tt *tester) checkConsistency() (failed bool, err error) {
 	}
 	plog.Printf("%s updated current revisions with %d", tt.logPrefix(), tt.currentRevision)
```

Comment:
this line should not be printed if the hash are not consistent. also if the hash is not consistent, there is no point to keep revision. the cluster will be cleaned up anyway.